### PR TITLE
Feature/plu 4060 add alphabetical sorting in class check

### PR DESCRIPTION
**Pull request by BOB41K1987**
https://eonx.atlassian.net/browse/PLU-4060

### CODE REVIEW

File: src/Rector/SortConstAlphabeticallyRector.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+use PhpParser\Node\Stmt\ClassLike;
+use Rector\Core\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Order\Contract\RankeableInterface;
+use Rector\Order\StmtOrder;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use Webmozart\Assert\Assert;
+
+final class SortConstAlphabeticallyRector extends AbstractRector implements ConfigurableRectorInterface
```
Comment (roman-eonx) [inline src/Rector/SortConstAlphabeticallyRector.php:20]: Let's use the full plural word, as in other cases: `Const` -> `Constants`. The same goes for the ValueObject.

**Review by roman-eonx — CHANGES_REQUESTED** [review]

**Review by alexndlm — APPROVED** [review]

File: src/Rector/SortConstAlphabeticallyRector.php (hunk 1 above)
Comment (BOB41K1987) [reply src/Rector/SortConstAlphabeticallyRector.php]: Done.

**Review by roman-eonx — APPROVED** [review]

File: src/Rector/EasyRankeable/ClassConstEasyRankeable.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+
+namespace EonX\EasyQuality\Rector\EasyRankeable;
+
+use PhpParser\Node\Stmt\ClassConst;
+use Rector\Order\Contract\RankeableInterface;
+
+final class ClassConstEasyRankeable implements RankeableInterface
```
Comment (akl-95) [inline src/Rector/EasyRankeable/ClassConstEasyRankeable.php:9]: What about `ClassConstant...` instead to correspond the other class names?

File: src/Rector/SortConstantsAlphabeticallyRector.php [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    }
+
+    /**
+     * @param RankeableInterface[] $rankeables
+     *
+     * @return string[]
+     */
+    private function sortByRanksAndGetNames(array $rankeables): array
+    {
+        \uasort($rankeables,
```
Comment (ERuban) [inline src/Rector/SortConstantsAlphabeticallyRector.php:105]: Please, move `$rankeables` to the new line. Here and in other such places.

File: src/Rector/ValueObject/SortConstantsAlphabetically.php [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+     * @var string
+     */
+    private $rankeableClass;
+
+    public function __construct(string $rankeableClass)
+    {
+        $this->rankeableClass = $rankeableClass;
+    }
+
+    public function getRankeableCLass(): string
```
Comment (akl-95) [inline src/Rector/ValueObject/SortConstantsAlphabetically.php:18]: Pls fix typo `CL` -> `Cl`.

**Review by akl-95 — CHANGES_REQUESTED** [review]

**Review by ERuban — CHANGES_REQUESTED** [review]

File: src/Rector/SortConstantsAlphabeticallyRector.php (hunk 3 above)
Comment (BOB41K1987) [reply src/Rector/SortConstantsAlphabeticallyRector.php]: Done.

File: src/Rector/ValueObject/SortConstantsAlphabetically.php (hunk 4 above)
Comment (BOB41K1987) [reply src/Rector/ValueObject/SortConstantsAlphabetically.php]: Done.

File: src/Rector/EasyRankeable/ClassConstEasyRankeable.php (hunk 2 above)
Comment (BOB41K1987) [reply src/Rector/EasyRankeable/ClassConstEasyRankeable.php]: Fixed.

**Review by ERuban — APPROVED** [review]

**Review by akl-95 — APPROVED** [review]

**Review by roman-eonx — APPROVED** [review]

**Pull request merged**